pkg/api/aim/runs.go

```diff
@@ -721,27 +707,29 @@ func SearchMetrics(c *fiber.Ctx) error {
 	tx := database.DB.
 		Select(`
 			metrics.*,
-			c.json AS context_json`,
+			runmetrics.context_json`,
 		).
 		Table("metrics").
 		Joins(
-			"INNER JOIN (?) runmetrics USING(run_uuid, key)",
+			"INNER JOIN (?) runmetrics USING(run_uuid, key, context_id)",
 			pq.Filter(database.DB.
 				Select(
 					"runs.run_uuid",
 					"runs.row_num",
 					"latest_metrics.key",
+					"latest_metrics.context_id",
+					"latest_metrics_context.json AS context_json",
 					fmt.Sprintf("(latest_metrics.last_iter + 1)/ %f AS interval", float32(q.Steps)),
 				).
 				Table("runs").
 				Joins(
 					"INNER JOIN experiments ON experiments.experiment_id = runs.experiment_id AND experiments.namespace_id = ?",
 					ns.ID,
 				).
-				Joins("LEFT JOIN metrics USING(run_uuid)").
-				Joins("LEFT JOIN latest_metrics USING(run_uuid)")),
+				Joins("LEFT JOIN latest_metrics USING(run_uuid)").
+				Joins(`LEFT JOIN contexts latest_metrics_context `+
+					`ON latest_metrics.context_id = latest_metrics_context.id`)),
```

Comment:
it is a bit confusing alias -> `latest_metrics_context`. why it can't be just nothing?

Reply:
the database was complaining about ambiguity at some point and required the alias, but I can see if subsequent change has resolved

Reply:
done

Reply:
ok, no problem.